applications/matter_bridge/README.rst

```diff
@@ -107,29 +118,58 @@ You can use the following commands to control the bridge device:
 
 * For adding a new bridged device to the bridge:
 
-   .. code-block:: console
+   * For simulated bridged device:
 
-      matter_bridge add <device_type> [node_label]
+      .. code-block:: console
 
-   This command adds a new bridged device to the bridge.
-   In this command:
+         matter_bridge add <device_type> [node_label]
 
-   * ``<device_type>`` is a Matter device type to use for the bridged device.
-     The argument is mandatory and accepts the following values:
+      This command adds a new bridged device to the bridge.
+      In this command:
 
-      * ``256`` - On/Off Light.
-      * ``770`` - Temperature Sensor.
-      * ``775`` - Humidity Sensor.
+      * ``<device_type>`` is a Matter device type to use for the bridged device.
+      The argument is mandatory and accepts the following values:
 
-   * ``[node label]`` is a node label for the bridged device.
-     The argument is optional and you can use it to better personalize the device in your application.
+         * ``256`` - On/Off Light.
+         * ``770`` - Temperature Sensor.
+         * ``775`` - Humidity Sensor.
 
```

Comment:
The numbers are device type ids from the Matter spec, so I thought it will be better to use them instead of creating some custom numbering. And the supported device types are displayed in the command help or if argument is not provided:

> uart:~$ matter_bridge add
add: wrong parameter count
add - Adds bridged device. 
      Usage: add <bridged_device_type> [node_label]
      * bridged_device_type - the bridged device's type, e.g. 256 - OnOff Light, 770 - TemperatureSensor, 775 - HumiditySensor
      * node_label - the optional bridged device's node label

Reply:
Oh, right, then everything is alright :)